source/CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 project(PasswordManager)
 
-add_executable(passwordManager main.cpp PasswordManager.cpp)
+add_library(sqlite3 STATIC ../lib/sqlite3.c)
+
+add_executable(passwordManager main.cpp PasswordManager.cpp DataBase/DataBaseHandler.cpp DataBase/DataBaseCommon.cpp)
```

Comment:
Instead of adding new sources with relative path add new CMakeLists.txt in DataBase folder. Try this:
https://stackoverflow.com/questions/8934295/add-source-in-a-subdirectory-to-a-cmake-project